deploy/examples/create-external-cluster-resources.py

```diff
@@ -825,12 +831,24 @@ def get_rbd_provisioner_caps_and_entity(self):
         }
         if self._arg_parser.restricted_auth_permission:
             rbd_pool_name = self._arg_parser.rbd_data_pool_name
+            alias_rbd_pool_name = self._arg_parser.alias_rbd_data_pool_name
             cluster_name = self._arg_parser.cluster_name
             if rbd_pool_name == "" or cluster_name == "":
                 raise ExecutionFailureException(
                     "mandatory flags not found, please set the '--rbd-data-pool-name', '--cluster-name' flags"
                 )
-            entity = f"{entity}-{cluster_name}-{rbd_pool_name}"
+            if rbd_pool_name.count(".") != 0 or rbd_pool_name.count("_") != 0:
```

Comment:
If there is some character besides period or underscore that we're not aware there is an issue, we need to allow an alias to be set. So I think we can cover that if we check here for the alias also, something like this?
```suggestion
            if rbd_pool_name.count(".") != 0 or rbd_pool_name.count("_") != 0 or alias_rbd_pool_name != "":
```

Reply:
Okay